packages/fiori/src/themes/BarcodeScannerDialog.css

```diff
@@ -0,0 +1,28 @@
+.ui5-barcode-scanner-dialog-video-wrapper,
+.ui5-barcode-scanner-dialog-video {
+    height:100%;
+    width: 100%;
+}
+
+.ui5-barcode-scanner-dialog-video {
+    object-fit: cover;
+}
+
+.ui5-barcode-scanner-dialog-footer {
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    padding-right: 0.5rem;

```

Comment:
check if in RTL you should mirror the padding with padding-left: 0.5rem

Reply:
Done